locations/spiders/vakif_katilim_tr.py

```diff
@@ -0,0 +1,30 @@
+import scrapy
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+
+
+class VakifKatilimTRSpider(scrapy.Spider):
+    name = "vakif_katilim_tr"
+    item_attributes = {"brand": "Vakıf Katılım", "brand_wikidata": "Q31188912"}
+    start_urls = ["https://www.vakifkatilim.com.tr/tr/diger/subeler-ve-atmler"]
+
+    def parse(self, response):
+        script_text = response.xpath('//script[contains(text(), "langId:")]//text()').get()
+        lang_id = scrapy.Selector(text=script_text).re_first(r"langId:\s*'([^']+)'")
```

Comment:
Does this need to be a `Selector`, or should we just do `re.search`?

Reply:
Thank you!